src/py21cmfast/inputs.py

```diff
@@ -108,7 +108,26 @@ class GlobalParams(StructInstanceWrapper):
         .. note:: If you are interested in snapshots of the same realization at several
                   redshifts,it is recommended to turn off this feature, as halos can
                   stochastically "pop in and out of" existence from one redshift to the next.
-
+    R_OVERLAP_FACTOR : float
+        When using USE_HALO_FIELD, it is used as a factor the halo's radius, R, so that the
+        effective radius is R_eff = R_OVERLAP_FACTOR * R.  Halos whose centers are less than
+        R_eff away from another halo are not allowed. R_OVERLAP_FACTOR = 1 is fully disjoint
+        R_OVERLAP_FACTOR = 0 means that centers are allowed to lay on the edges of
+        neighboring halos.
+    DELTA_CRIT_MODE : int
+        The delta_crit to be used for determining whether a halo exists in a cell
+            0: delta_crit is constant (i.e. 1.68)
+            1: delta_crit is the sheth tormen ellipsoidal collapse correction to delta_crit
+    HALO_FILTER : int
+        Filter for the density field used to generate the halo field with EPS
+            0: real space top hat filter
+            1: sharp k-space filter
+            2: gaussian filter
+    OPTIMIZE : int
+        Finding halos can be made more efficient if the filter size is sufficiently large that
```

Comment:
I would tend towards including these in `UserParams`, so that they're easier to discover, and a bit more flexible. However, we could also opt for doing that later with all of the global params.